index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Singolo</title>
+    <link rel="stylesheet" href="my_css.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="header_wrapp">
+            <div class="wrapp_logo_container">
+                <h1 class="h1_wrapp">singolo<span class="h1_wrapp_logo">*</span></h1>
```

Comment:
Please try to add more sense to the names of classes and make them as informative as possible for the developer.
See more example here https://ru.bem.info/methodology/naming-convention/
https://dev.to/ziizium/css-naming-conventions-5gd6